Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/crash.ui/SettingsCrashReportFragment.kt

```diff
@@ -0,0 +1,68 @@
+package de.rki.coronawarnapp.test.crash
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.bugreporting.event.BugEvent
+import de.rki.coronawarnapp.databinding.FragmentCrashreporterOverviewBinding
+import de.rki.coronawarnapp.test.menu.ui.TestMenuItem
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import timber.log.Timber
+import javax.inject.Inject
+
+class SettingsCrashReportFragment : Fragment(R.layout.fragment_crashreporter_overview), AutoInject,
+    CrashReportAdapter.ItemClickListener {
```

Comment:
Prefer inline listeners instead of global interface implementations on a Fragment, smaller scope, easier to see what is going on.

Reply:
done.